### PR TITLE
Fix minor updates so that the template properly resets

### PR DESCRIPTION
**Pull request by CoderDake**
While prepping a branch to update us to a dev version on the master branch I noticed that the NEXT_RELEASE template wasn't properly updating. To fix this I've added linting rules that will make sure we always await our futures.

### CODE REVIEW

File: tool/update_version.dart [hunk 1]
```diff
@@ -0,0 +1,388 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
```
Comment (kenzieschmoll) [inline tool/update_version.dart:1]: why was this file moved out of commands?

Comment (CoderDake) [reply tool/update_version.dart:1]: oh shoot this is an artifact I forgot to push up my remove

**Review by kenzieschmoll — APPROVED** [review]
